clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -36,8 +36,13 @@ include "mlir/IR/SymbolInterfaces.td"
 // CIR Ops
 //===----------------------------------------------------------------------===//
 
+class LoweringInfo {
```

Comment:
Do you plan to use this for things other than facilitating LLVM lowering? If not, I suggest you rename this to `LLVMLoweringInfo` for now.